service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/LanguageRequirementsCheckerTest.java

```diff
@@ -2,30 +2,37 @@
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static uk.gov.hmcts.reform.fpl.model.tasklist.TaskState.COMPLETED_FINISHED;
 
-@ExtendWith(MockitoExtension.class)
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = LanguageRequirementsChecker.class)
```

Comment:
Don't need either of these, stop finding ways to add annotations to tests 😛

Reply:
the mock injection or annotations are needed to avoid null pointer exceptions, will run through it tomorrow if you want.

Reply:
Sure thing, I wouldn't expect it to be needed

Reply:
Updated